pkg/operator/cluster/ceph/mon/health.go

```diff
@@ -126,6 +127,8 @@ func (c *Cluster) checkHealth() error {
 			if time.Since(c.monTimeoutList[mon.Name]) <= MonOutTimeout {
 				logger.Warningf("mon %s NOT found in quorum, STILL in mon out timeout", mon.Name)
```

Comment:
while it's still just a warning, could we NOT use caps? :) People get worried about caps.